api/src/main/java/io/opentelemetry/metrics/DoubleSumObserver.java

```diff
@@ -67,15 +76,12 @@
     @Override
     Builder setConstantLabels(Map<String, String> constantLabels);
 
-    @Override
-    Builder setMonotonic(boolean monotonic);
-
     @Override
     DoubleSumObserver build();
   }
 
   /** The result for the {@link AsynchronousInstrument.Callback}. */
-  interface ResultDoubleObserver {
+  interface ResultDoubleSumObserver {
```

Comment:
Not for this PR, but it might read a little better if the `Result` word were at the end, rather than the beginning of the interface names.

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/1230